cmd/auth/auth.go

```diff
@@ -34,25 +37,29 @@ GCP: https://docs.gcp.databricks.com/en/dev-tools/auth/index.html`,
 }
 
 func promptForHost(ctx context.Context) (string, error) {
+	if !cmdio.IsInTTY(ctx) {
+		return "", fmt.Errorf("the command is being run in a non-interactive environment, please specify a host using --host")
+	}
+
 	prompt := cmdio.Prompt(ctx)
-	prompt.Label = "Databricks Host (e.g. https://<databricks-instance>.cloud.databricks.com)"
-	// Validate?
-	host, err := prompt.Run()
-	if err != nil {
-		return "", err
+	prompt.Label = "Databricks Host"
```

Comment:
```suggestion
	prompt.Label = "Databricks workspace URL"
```

Reply:
The "auth login" command also works for account authentication.

Reply:
So workspace our account URL? That will be a bit more familiar to customers. And it's not really a host if it includes https://